### DISCUSSION TITLE
installing fpm on windows

### DISCUSSION THREAD

**Discussion by dpapantoniou** [fortran-lang/fpm#673] [General]
Could someone help with insructions how to download and install fpm on a windows-10 system?
Is fpm compatible with the minGW f90 compiler?

**Comment by awvwgk** [comment]
There are several options for you here, please checkout the installation instructions marked with the Windows logo or follow the instructions in the compile from source category at: https://fpm.fortran-lang.org/en/how-to/installation.html.
